third-party-programs.txt

```diff
@@ -0,0 +1,570 @@
+Unified Memory Framework (UMF) Third Party Programs File
+
+This file is the "third-party-programs.txt" file specified in the associated
+Intel end user license agreement for the Intel software you are licensing.
+
+The third party programs and their corresponding required notices and/or
+license terms are listed below.
+_______________________________________________________________________________
+
+1.  Modified mimaloc new/delete implementation:
+
+    MIT License
+
+    Copyright (c) 2018-2021 Microsoft Corporation, Daan Leijen
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in
+    all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+
+_______________________________________________________________________________
+
+2.  Portable Hardware Locality (hwloc):
```

Comment:
Do we copy, or distribute hwloc? if not we don't need it here

Reply:
there is an option where hwloc is statically linked with UMF (UMF_LINK_HWLOC_STATICALLY), so yes this is needed